README.rst

```diff
@@ -38,3 +38,21 @@ coala supports popular programming languages including **Python**, **C/C++**, **
 .. image:: https://graphs.waffle.io/coala/coala/throughput.svg
  :target: https://waffle.io/coala/coala/metrics/throughput
  :alt: 'Throughput Graph'
+
+Using ``default_coafile``
+
+* ``default_coafile`` is a normal coala configuration
+  file with system wide effect.
+
+* It can be used to define the type of files you definitely
+  don't want to lint like minified files
+  and backup files ( e.g. ``*.min.js``, ``*.orig``).
+
+* Any modifications made in the ``default_coafile`` will be
```

Comment:
As you mentioned two paths to `default_coafile` so when you are talking about modification then which one you are pointing out,if you explain this thing then it would be more clear.